Add scroll-driven animations article (#6147)
GoogleChrome/developer.chrome.com: site/en/articles/scroll-driven-animations/index.md

---
layout: 'layouts/blog-post.njk'
title: Animate elements on scroll with Scroll-driven animations
authors:
  - bramus
subhead: >
  Learn how to work with Scroll Timelines and View Timelines to create scroll-driven animations in a declarative way.
description: >
  Learn how to work with Scroll Timelines and View Timelines to create scroll-driven animations in a declarative way.
date: 2023-05-10
hero: image/AeNB0cHNDkYPUYzDuv8gInYA9rY2/iDNlcxHa6axrfgHi4aoU.png

> alt: "Illustration to introduce “Scroll-driven Animations”. It pictures a mouse and an arrow indicating scroll movement."

Reviewer: ```suggestion
alt: "A computer mouse and an arrow indicating scroll movement."
```
